worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/CharSetBlocks.java

```diff
@@ -330,9 +330,21 @@ public boolean hasBiomes(int layer) {
     }
 
     @Override
-    public char[] load(final int layer) {
+    public char[] loadChars(final int layer) {
         updateSectionIndexRange(layer);
-        return super.load(layer);
+        return super.loadChars(layer);
+    }
+
+    @Override
+    public int[] loadInts(final int layer) {
+        throw new UnsupportedOperationException("The class CharSetBlocks does not support the usage of loadInts(). Please " +
+                "report this error to the FastAsyncWorldEdit developers.");
+    }
+
+    @Override
+    public int[] loadIntsIfPresent(final int layer) {
```

Comment:
This method ought to be wherever `loadCharsIfPresent` is located in the hierarchy